src/poll/ShowPoll.js

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from 'react'
+import {Link} from 'react-router'
+import {find} from 'lodash'
+import {get} from '../helpers.js'
+import Loader from './Loader'
+
+import css from './ShowPoll.css'
+
+class ShowPoll extends Component{
+  state = {
+    poll: {},
+    totalVotes: 0,
+  }
+  componentWillMount() {
+    get(`poll/${this.props.params.id}/results`).then((response)=> {
+      document.title = `${response.data.title} – Results`
+      this.setState({
+        poll: Object.assign({}, response.data),
+        totalVotes: this.getVotesTotal(response.data.votes),
+      })
+    })
+
+  }
+  render() {
+    if(!this.state.poll.id){
```

Comment:
This is ok for this exercise. In the future, you could consider keeping the status of the HTTP request in the component state. So you'd set `this.setState({isLoading: true})` upon starting the GET request, and setting it back to false when it has finished fetching the data.